src/mo_types/arrange_type.ml

```diff
@@ -43,6 +43,19 @@ let prim = function
 
 let con c = Atom (Type.string_of_con c)
 
+let pos p =
+  "Pos" $$
+    [ Atom p.Source.file
```

Comment:
This will lead to lots of duplicated file names in the s-expression (twice for every region, and then many regions might share the same filename)

Reply:
I've changed it so the `region` has the filename from the left position, and removed it from `pos`. Note that in `Arrange`, the filename is introduced in both `pos`itions, but if it matches the `main_file` from the configuration, then it outputs the empty string instead. I don't do the `main_file` check here, but if you'd like, I can change it. Since those are only output when there are source fields, I think it's fine.

(Note: I'm addressing this in https://github.com/serokell/motoko/pull/57)